packages/instantsearch.js/src/widgets/frequently-bought-together/__tests__/frequently-bought-together.test.tsx

```diff
@@ -313,40 +213,31 @@ describe('frequentlyBoughtTogether', () => {
                 <li
                   class="ais-FrequentlyBoughtTogether-item"
                 >
-                  <h2>
-                    Moschino Love – Shoulder bag
-                  </h2>
-                  <p>
-                    Moschino Love
-                  </p>
+                  <h2 />
+                  <p />
```

Comment:
Why are those empty?

Reply:
Oh I see, I changed the mocked data to reduce the amount of unnecessary elements in the dataset, but forgot to adjust the templates 🤦 ...